<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMeshSize</class>
 <widget class="QMainWindow" name="frmMeshSize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>局部尺寸</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>网格尺寸(m)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMaxSize">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_used">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>启用</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
